tests/codegen/ccode/scripts/arrays_slicing.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=missing-function-docstring, missing-module-docstring/
+# pylint: disable=missing-function-docstring, missing-module-docstring, unused-variable, undefined-variable/
```

Comment:
Here you can leave `unused-variable` disabled as there is no way to create a test which uses all variables with what is currently implemented

Reply:
This is also true in `arrays_create`, but there should be no reason to disable `undefined-variable`

Reply:
the undefined variables is for the pyccel decorators

Reply:
Search for examples of use of the decorator in other folders. Undefined-variable shouldn't be being raised